### PR TITLE
Resolved problems with pull request

### PR DESCRIPTION
**Pull request by Ramizjon**
@btrofimov new pull request

### CODE REVIEW

**Comment by btrofimov** [comment]
@Ramizjon - next time please specify proper PR name and correpsonding message details on what you have done ;)

**Comment by btrofimov** [comment]
bty why `.classpath`  is in PR ?

File: build.gradle [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+eclipse {
+     //Ensure Eclipse build output appears in build directory
+    classpath {
+        defaultOutputDir = file("${buildDir}/eclipse-classes")
+    }
+}
+
+Experience in development of Java applications for Android platform, and using Hadoop API. Familiar with Cloudera Distributed Hadoop and it's main modules. Knowledge of MySQL database management system, HBase and Hive. Understanding main concepts of MapReduce framework. Ability to learn new technologies and work in team.Good spoken english.
+
+Маришева Олександра - Місце: 024, СТУД СК09693694
```
Comment (btrofimov) [inline build.gradle]: ??? :)

File: src/main/java/logic/Main.java [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+import parquet.hadoop.example.GroupReadSupport;
+import parquet.schema.*;
+import parquet.tools.read.SimpleReadSupport;
+import parquet.tools.read.SimpleRecord;
+import parquet.tools.read.SimpleRecordMaterializer;
+
+public class Main{
+	
+	
+	public static void main(String[] args) throws Exception {
```
Comment (btrofimov) [inline src/main/java/logic/Main.java]: Better optino is to use `ToolRunner.run` from org.apache.hadoop.util.{ Tool, ToolRunner}
Also please isolate job creating into separate method

File: src/main/java/IO/UserModPrimitiveConverter.java [hunk 3]
```diff
@@ -0,0 +1,71 @@
+package IO;
```
Comment (btrofimov) [inline src/main/java/IO/UserModPrimitiveConverter.java]: wrong package name IO
Also pelase remove all files that are not related to csv format. I think they can be easily added back if needed.
Also pelase move usererpository to domain package and move DT object to logic package
Please rename logic package to something more meaningful like mapreduce or so.

**Comment by btrofimov** [comment]
@Ramizjon 
Well let's make a plan on what has to be improved:
- add gradle project file
- Classes should not use direct singletones. Instead, let's use fabric method pathern.
- App main class should use Apache Hadoop Tool class and correpsonding infrastructure, as discussed.
- what if mpreduce fails, shall the application fail with return code as well?

File: src/main/java/domain/UserRepository.java [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+	
+	public void checkForBulk(){
+		if (cachedList.size()==BUFFER_SIZE){
+			this.addUsersToHbase();
+			cachedList.clear();
+		}
+	}
+	
+	//just for debug
+	public String print(){
```
Comment (btrofimov) [inline src/main/java/domain/UserRepository.java:55]: This logic should be outside of this class.

File: src/main/java/domain/UserRepository.java [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+		Put put = null;
+		for (User u: cachedList){
+			put = new Put(Bytes.toBytes("Id" + String.valueOf(u.getUserId())));
+			for (String segm: u.getSegments()){
+				String timeStamp = new SimpleDateFormat("yyyy/MM/dd HH:mm:ss").format(System.currentTimeMillis());
+				put.add(Bytes.toBytes("general"),Bytes.toBytes(segm),Bytes.toBytes(timeStamp));
+				try {
+					hTable.put(put);
+				} catch (IOException e) {
+					e.printStackTrace();
```
Comment (btrofimov) [inline src/main/java/domain/UserRepository.java:73]: Do not dismiss such kind of error and remove printStackTrace at all from here. Moreover Repository _must_ throw up any excpeptions and only upstream logic should decide if this particular exception shall be dismissed.

File: src/main/java/domain/UserRepository.java [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+	
+	public void removeUserFromHbase(String rowId, LinkedList<String> segments){
+		Delete delete = new Delete (Bytes.toBytes(rowId));
+		for (String s: segments){
+			delete.deleteColumns(Bytes.toBytes("general"), Bytes.toBytes(s));
+		}
+		try {
+			hTable.delete(delete);
+		} catch (IOException e) {
+			e.printStackTrace();
```
Comment (btrofimov) [inline src/main/java/domain/UserRepository.java:88]: the same, see above

File: src/main/java/logging/Utilities.java [hunk 7]
```diff
@@ -0,0 +1,16 @@
+package logging;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class Utilities {
+	public final Logger logger = LoggerFactory.getLogger( "AnyUniqueStringHere" );
```
Comment (btrofimov) [inline src/main/java/logging/Utilities.java:7]: remove this class from the project. we disc ussed to use logger directly inside every class where it is required.

**Comment by Ramizjon** [comment]
Pull request is deprecated
